src/code-templates/services/order-service/entry-points/api/server.ts

```diff
@@ -7,16 +7,28 @@ import { defineRoutes } from "./routes";
 import { errorHandler } from "@practica/error-handling";
 import * as configurationProvider from "@practica/configuration-provider";
 import configurationSchema from "../../config";
+import { AsyncLocalStorage } from 'async_hooks';
+import { v4 } from 'uuid';
 
 let connection: Server;
+const asyncLocalStorage = new AsyncLocalStorage<Map<string, string>>();
+
+function requestId(req, res, next) {
```

Comment:
Since it's a cross Microservice thing, and we also need to put somewhere a global 'new AsyncLocalStorage' - Shall we make this a library?